var/spack/repos/builtin/packages/h5hut/package.py

```diff
@@ -47,7 +47,11 @@ def validate(self):
 
     def flag_handler(self, name, flags):
         build_system_flags = []
-        if name == "cflags" and self.spec["hdf5"].satisfies("@1.12:"):
+        if (
+            name == "cflags"
+            and self.spec.satisfies("@:1")
```

Comment:
Minor: I suppose this part could in theory be dropped now if version < 2 isn't a thing. Not sure what's right or if it can be removed.

Reply:
IMHO it's anyway good for documentation.